### DISCUSSION TITLE
Sample multianimal 3D data

### DISCUSSION THREAD

**Discussion by delaroob** [talmolab/sleap#2013] [Help!]
Hi!

I'm looking for a single piece of **points3d.h5** file, where 2 animals were tracked in 3D using sleap(-anipose, with that exact h5 file structure) and the data is _definitely_ good. 
Would anyone happen to have one lying around? I don't even need the videos or .slp files for it, I just have an R script I want to check: I don't know if the script isn't working, or if there was a problem with my calibration/+triangulation/+points3d.h5 file.

**Comment by talmo** [comment]
Hi @delaroob,

Would the data in [our tests here](https://github.com/talmolab/sleap-anipose/tree/aa6e68f27f2f7a64163c0739c17c8cf58837036f/tests/data/minimal_session) work?

Cheers,

Talmo

**Reply by delaroob** [reply]
I hope so, thanks!

**Reply by delaroob** [reply]
Okay, maybe not:D

(in tests\data\minimal_session\points3d.h5)

```
  group   name       otype dclass              dim
0     / tracks H5I_DATASET  FLOAT 3 x 15 x 1 x 120
```

I guess 3 is for x/y/z, 15 is the number of bodyparts, 1 is the number of instances and 120 is the number of frames.
(And I also checked out the linked videos, and there's just one mouse hanging out.)

I'm looking for something with (at least, but preferably exactly) 2 animals.
